src/main/components/PianoRoll/PianoRollEditor.tsx

```diff
@@ -1,21 +1,36 @@
+import { observer } from "mobx-react-lite"
 import { FC } from "react"
 import styled from "styled-components"
+import { useStores } from "../../hooks/useStores"
 import { PianoRollKeyboardShortcut } from "../KeyboardShortcut/PianoRollKeyboardShortcut"
 import { PianoRollToolbar } from "../PianoRollToolbar/PianoRollToolbar"
+import { TrackList } from "../TrackList/TrackList"
 import PianoRoll from "./PianoRoll"
 
-const Container = styled.div`
+const ColumnContainer = styled.div`
   display: flex;
   flex-direction: column;
   flex-grow: 1;
 `
 
-export const PianoRollEditor: FC = () => {
+const RowContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
```

Comment:
When there are many tracks in the list, view size will get bigger than window.
We should make TrackList scrollable.

![image](https://user-images.githubusercontent.com/5355966/134755424-a4b2ad10-da58-465a-af82-839a84a37b53.png)

Add styles to restrict RowContainer`s height.

```suggestion
  flex-grow: 1;
  overflow: hidden;
  flex-basis: 0;
```

Then add overflow style to TrackList.
https://github.com/ryohey/signal/blob/c5f56726066334cdba76d7f90f7decdde7c6cdb2/src/main/components/TrackList/TrackList.tsx#L86

```diff
-<List>
+<List style={{ overflowY: "auto" }}>
```

![image](https://user-images.githubusercontent.com/5355966/134755411-d005876b-8c64-4e6d-9976-4b6caa84194e.png)

Reply:
Great catch! Didn't test it with any songs that had that many tracks before you mentioned it. Updated based on your comments now. I must say that I don't think that scroll bar looks very pretty (I would be nicer if it were more themed like the other scroll bars), but I guess the style can be updated in a new PR